### ISSUE TITLE
[Main UI - Color Widgets] Widgets for Extended Color Lights are not shown

### ISSUE THREAD

**Issue by curlyel** [openhab/openhab-webui#929]
## The problem

Color items for newly discovered things (Extended Color Lights) don't get any color widget shown in all parts of the Main UI. Eg.:
![image](https://user-images.githubusercontent.com/11849305/109276817-c7b3e000-7816-11eb-929a-5229269cc036.png

The behaviour has been seen so far with lights discovered by the Deconz binding as well as by the Zigbee binding.
For the Deconz binding some change seemed to solve the issue (see: https://github.com/openhab/openhab-addons/issues/10107#)

I already opened an issue in the openhab-addon repository related to the Zigbee binding (see: https://github.com/openhab/openhab-addons/issues/10223)
Although, @cdjackson as the maintainer of the Zigbee binding suspects the cause in the webui. Therefore I raise this issue here too to start the discussion if it is to be solved in the binding _or_ the webui.

The following seem to support Chris's point of view:

```
openhab> send ZIGRGBWStripBuero_Color ON
Command has been sent successfully.
openhab> send ZIGRGBWStripBuero_Color 120,75,41
Command has been sent successfully.
openhab>
```

Openhab accepts the ON/OFF as well as the color commands sent to the item. And the real device indeed updates the color.

## Expected behavior

I'd expect a color widget show in standalone or lists allowing to control the color, e.g.:
![image](https://user-images.githubusercontent.com/11849305/109278514-e74c0800-7818-11eb-876e-a05913274f52.png)

## Steps to reproduce

- discover extended color light by the Zigbee binding (maybe other binding are affected as well)
- bind the color channel of the discovered light to a color item by "create equipment from thing" from within the model page
- 
## Your environment

```yaml
runtimeInfo:
  version: 3.1.0.M1
  buildString: Milestone Build
locale: de_DE
systemInfo:
  configFolder: /etc/openhab
  userdataFolder: /var/lib/openhab
  logFolder: /var/log/openhab
  javaVersion: 11.0.10
  javaVendor: Azul Systems, Inc.
  javaVendorVersion: Zulu11.45+27-CA
  osName: Linux
  osVersion: 5.10.12-sunxi
  osArchitecture: arm
  availableProcessors: 4
  freeMemory: 105461296
  totalMemory: 323035136
bindings:
  - astro
  - bluetooth
  - harmonyhub
  - hue
  - knx
  - kodi
  - miio
  - mqtt
  - network
  - networkupstools
  - rfxcom
  - shelly
  - sonos
  - systeminfo
  - zigbee
  - zwave
clientInfo:
  device:
    ios: false
    android: false
    androidChrome: false
    desktop: true
    iphone: false
    ipod: false
    ipad: false
    edge: false
    ie: false
    firefox: false
    macos: false
    windows: true
    cordova: false
    phonegap: false
    electron: false
    nwjs: false
    webView: false
    webview: false
    standalone: false
    os: windows
    pixelRatio: 1.25
    prefersColorScheme: dark
  isSecureContext: false
  locationbarVisible: true
  menubarVisible: true
  navigator:
    cookieEnabled: true
    deviceMemory: N/A
    hardwareConcurrency: 8
    language: de-DE
    languages:
      - de-DE
      - de
      - en-US
      - en
    onLine: true
    platform: Win32
  screen:
    width: 3072
    height: 1728
    colorDepth: 24
  support:
    touch: false
    pointerEvents: true
    observer: true
    passiveListener: true
    gestures: false
    intersectionObserver: true
  themeOptions:
    dark: light
    filled: true
    pageTransitionAnimation: default
    bars: filled
    homeNavbar: default
    homeBackground: default
    expandableCardAnimation: default
  userAgent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML,
    like Gecko) Chrome/88.0.4324.182 Safari/537.36
timestamp: 2021-02-26T08:59:41.316Z


```

**Comment by curlyel** [comment]
More testing showed me, although the command got accepted (and the color of the actual device changed accordingly), the items state remains UNDEF, notice last line:

```
openhab> send ZIGRGBWStripBuero_Color 120,75,45
Command has been sent successfully.
openhab> items list ZIGRGBWStripBuero_Color
ZIGRGBWStripBuero_Color (Type=ColorItem, State=UNDEF, Label=Color, Category=ColorLight, Tags=[Point], Groups=[ZIGRGBWStripBuero])

```
This **may indicate**, that the reason for the issue is rather on the binding or the core than on the ui side though … or?

**Comment by stefan-hoehn** [comment]
I discovered a similar behaviour in my nanoleaf binding:

- I create a color item for the main controller and it reveals a color picker like shown above
- I create a color item for a thing that is bound to the controller bridge and does not show the color picker.

If I create entries in a sitemap for both it does work as expected. If you need more details don't hesitate to come back to me.

**Comment by curlyel** [comment]
Seems to be a UI issue according to Yannicks statement here:

https://community.openhab.org/t/colorpicker-in-oh3/110733/8?u=curlyel

**Comment by Sandmann-cs1** [comment]
The bug is also discussed here
https://community.openhab.org/t/zigbee-binding-color-not-working/124868

Would be nice if someone could make the color widget able to handle UNDEF state.

Thanx

**Comment by Skinah** [comment]
Bug also discussed here
https://community.openhab.org/t/colorpicker-widget-not-displaying-anything/125375

**Comment by florian-h05** [comment]
Fixed by #2270.
